### PR TITLE
Added CentOS repos for 3.1

### PR DESCRIPTION
**Pull request by Bischoff**
We were putting 3.2 repos on 3.1 CentOS.

I am not sure at all about the difference between
  `Devel_Galaxy_Manager_3.2_RES-Manager-Tools-7-x86_64.repo`
and
  `Devel_Galaxy_Manager_3.2_RES-Manager-Tools-7-Beta-x86_64.repo`.

I suspect an error there. Cen you please review that too?

### CODE REVIEW

**Comment by moio** [comment]
Client repos are just one version per OS, irrespective of the Manager version. We might have a different version for Uyuni.

**Comment by Bischoff** [comment]
Isn't that covered by 'Head`?

Honestly, I'm a bit lost here. Can you please tell me exactly what I should change?

**Comment by Bischoff** [comment]
I humbly suggest we drop this PR, that you do the PR exactly how you want it, and that I adapt in the testsuite.

**Comment by Bischoff** [comment]
Here is what we have in the various branches of the test suite:
```
     When I enable repository "Devel_Galaxy_Manager_3.1_RES-Manager-Tools-7-x86_64" on this "ceos-traditional-client"
     When I enable repository "Devel_Galaxy_Manager_3.2_RES-Manager-Tools-7-x86_64" on this "ceos-traditional-client"
     When I enable repository "Devel_Galaxy_Manager_Head_RES-Manager-Tools-7-x86_64" on this "ceos-traditional-client"
```

**Comment by MalloZup** [comment]
@moio : 

1.) This should be removed 
https://github.com/moio/sumaform/blob/master/salt/repos/default.sls#L324

especially the `beta` for a released tag. https://github.com/moio/sumaform/blob/master/salt/repos/default.sls#L327.

It should be  https://github.com/moio/sumaform/blob/master/salt/repos/default.sls#L345
( i think this should be adressed on other PRS ( i think all the `released` tag are affected)

2) @Bischoff  what moio mean here is:

the client tools in SUSE-Manager are valid for all SLE12 family clients.

so having https://github.com/moio/sumaform/blob/master/salt/repos/default.sls#L342
we can alllow `3.1-nightly`, `3.2-nightly` etc.

In this centos7 case, we don\t need other repos https://github.com/moio/sumaform/blob/master/salt/repos/default.sls#L306, because this tool are ok

File: salt/repos/default.sls [hunk 1]
```diff
@@ -330,23 +330,42 @@ tools_update_repo:
...
+    - source: salt://repos/repos.d/Devel_Galaxy_Manager_3.1_RES-Manager-Tools-7-x86_64.repo
     - template: jinja
     - require:
       - cmd: galaxy_key
 
-{% elif 'nightly' in grains.get('product_version') | default('', true) %}
+{% elif '3.2-released' in grains.get('product_version') | default('', true) %}
+tools_update_repo:
+  file.managed:
+    - name: /etc/yum.repos.d/SLE-Manager-Tools-RES-7-Beta-x86_64.repo
```
Comment (MalloZup) [inline salt/repos/default.sls]: released beta :rabbit: 

I think this need a separate PR @moio for checking all this

**Comment by Bischoff** [comment]
@moio @MalloZup would it be acceptable to merge this PR "as is" to fix the 3.1 production testsuite, and then to refine in a separate PR?

**Comment by MalloZup** [comment]
@Bischoff  to me, i am thinking that the `client tool model doesn`t apply for centos7 but only for SLE12.

I would ping @juliogonzalez or @mcalmer  for this.

For the `released beta` bug open an issue

**Comment by Bischoff** [comment]
I only copied and pasted `-Beta`lines. I am unable to judge of their relevance. Better discuss and fix it in a separate PR. I created issue #422 for that.

**Comment by Bischoff** [comment]
I needed 3.1 to be green again. I think it's clean enough to be reworked. I merged.

**Pull request merged**